e2e/config.js

```diff
@@ -6,9 +6,11 @@
  */
 
 module.exports = {
+  // TODO: Change back to https://scaffold-pwa-e2e-tests-pwa-kit.mobify-storefront.com
+  // when v3 is officially released
```

Comment:
Just to make it clear which v3 version is this for.
```suggestion
  // TODO: Change back to https://scaffold-pwa-e2e-tests-pwa-kit.mobify-storefront.com
  // when retail-react-app v3 is officially released
```